ddtrace/llmobs/_writer.py

```diff
@@ -67,10 +70,25 @@ class LLMObsEvaluationMetricEvent(TypedDict, total=False):
     tags: List[str]
 
 
+def configure_agentless_enabled() -> None:
+    """Configure whether to use agentless mode based on agent availability and capabilities."""
+    if config._llmobs_agentless_enabled is not None:
+        return
+
+    agent_info = agent.info()
+
+    if agent_info is None:
+        config._llmobs_agentless_enabled = True
+        return
+
+    endpoints = agent_info["endpoints"]
+    config._llmobs_agentless_enabled = "/evp_proxy/v2/" not in endpoints
```

Comment:
Instead of setting the config value here, let's do that in `LLMObs.enable()` and make this method return whether or not we should be using agentless enabled.

Reply:
makes sense to me! pushed